README.md

```diff
@@ -50,6 +50,12 @@ To help anyone with any level of familiarity with Jekyll quickly get started, Po
 
 Folks wishing to use Jekyll's templates and styles can do so with a little bit of manual labor. Download Poole and then copy what you need (likely `_layouts/`, `*.html` files, `atom.xml` for RSS, and `public/` for CSS, JS, etc.).
 
+### 2c. Add gems
+
+To add gems update the _config.yml in the format
+gems:
+- <gem-name>
```

Comment:
This will not render correctly, as is.
1. The string "_config.yml" should be wrapped in back-ticks to show that it's a filename: `_config.yml`
1. The `gems` block should be wrapped in triple back-ticks to render as code:

~~~
```yaml
gems:
  - <gem-name>
```
~~~